signing/src/main/java/tech/pegasys/web3signer/signing/config/metadata/AzureSecretSigningMetadata.java

```diff
@@ -87,4 +90,11 @@ public String getSecretName() {
   public ArtifactSigner createSigner(final ArtifactSignerFactory factory) {
     return factory.create(this);
   }
+
+  @Override
+  public Map<String, String> getTags() {
+    // tags support is not applicable for config file mode as
+    // user is already providing the secret name to load the secret from.
+    return Collections.emptyMap();
```

Comment:
Could/should we throw unimplemented exception here? Feels like we should log a warning too.